iris/meta.yaml

```diff
@@ -26,13 +25,12 @@ requirements:
         - setuptools
         - mo_pack  # [not win]
         - nc_time_axis
-        - iris_grib  # [not win]
+        - iris_grib  # [not win or not py3k]
```

Comment:
This condition is wrong, or will include iris_grib as a dependenciy if one of the conditions is true, meaning if we are not on Windows (not win == True) then we will include, regardless of the value of py3k. An `and` should be used instead of `or` I believe.